free5gc-cp/amf/amf-deployment.yaml

```diff
@@ -0,0 +1,74 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: free5gc-amf
+  labels:
+    app.kubernetes.io/version: "v3.1.1"
+    project: free5gc
+    nf: amf
+  annotations:
+    config.kubernetes.io/local-config: "true"
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      project: free5gc
+      nf: amf
+  template:
+    metadata:
+      annotations:
+        k8s.v1.cni.cncf.io/networks: '[
+            {
+              "name": "n2network-amf",
+              "interface": "n2",
+              "ips": [ "10.100.50.249/29" ],
```

Comment:
Will these IPs be specialized to an environment? ( AFAIK we are not specializing these packages)

Reply:
the N2 network for AMF will be specialized. The SBA interface not of AMF will not be specialized

Reply:
AMF and SMF are here just for reference. They are marked with `config.kubernetes.io/local-config: "true"` annotation, which means that all AMF and SMF KRM resources will be omitted.